Cargo.toml

```diff
@@ -142,29 +147,33 @@ ethers-providers = { version = "2.0", default-features = false }
-reth-tasks = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.5" }
-reth-db = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.5" }
-reth-trie = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.5" }
-reth-rpc = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.5" }
-reth-stages = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.5" }
+revm-inspectors = { git = "https://github.com/paradigmxyz/evm-inspectors", rev = "7168ac5", default-features = false }
+reth-primitives = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
+reth-interfaces = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
+reth-rpc-types = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
+reth-rpc-types-compat = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
+reth-node-api = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
+reth-transaction-pool = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
+reth-provider = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
+reth-tasks = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
+reth-db = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
+reth-trie = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
+reth-rpc = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
+reth-stages = { git = "https://github.com/paradigmxyz/reth", tag = "v0.2.0-beta.7", default-features = false }
 
-revm = { version = "7.2.0", features = ["serde"], default-features = false }
+revm = { version = "8", features = ["serde"], default-features = false }
 # forcing cargo for this version or else chooses 3.1.1 and there is some dependency conflicts
-revm-primitives = { version = "=3.1.0", default-features = false }
+revm-primitives = { version = "3.1", default-features = false }
 # forcing cargo for this version or else chooses 0.3.1 and there is some dependency conflicts
 alloy-trie = { version = "=0.3.0", default-features = false }
-alloy-primitives = "0.6.4"
-alloy-sol-types = "0.6.4"
+alloy-rlp = { version = "0.3", default-features = false }
+alloy-primitives = { version = "0.7.4", default-features = false }
+alloy-sol-types = { version = "0.7.4", default-features = false }
 
-secp256k1 = { version = "0.27.0", default-features = false, features = [
+ed25519-dalek = { version = "2", default-features = false, features = [
+    "serde",
+] }
+secp256k1 = { version = "0.28.2", default-features = false, features = [
```

Comment:
is the patch defined below for secp256k1 still used?
given that this is 28.2 and thats 28?

Reply:
Yes